packages/serialize/src/index.js

```diff
@@ -304,6 +309,15 @@ export const serializeStyles = function(
     stringMode = false
     styles += handleInterpolation(mergedProps, registered, strings, false)
   } else {
+    if (
+      process.env.NODE_ENV !== 'production' &&
+      strings.length !== args.length
+    ) {
+      console.warn(
+        `\`css\` got used in confusing manner - most likely you have forwarded all arguments from tagged template call to \`css\` with extra stuff appended to them.` +
+          ` This may cause parsing problems. Like in this example code snippet:\n${TAGGED_TEMPLATE_ARGS_WEIRDLY_FORWARDED_SNIPPET}`
```

Comment:
- this isn't necessarily being called from css, it could be styled or others
- confusing seems like a strange word choice
- `This may cause parsing problems` - this isn't really about parsing 
- the message doesn't say what people should do to fix it
- I don't trust tools to eliminate the code example outside of `process.env.NODE_ENV !== 'production'`